src/main/jbake/templates/index.ftl

```diff
@@ -5,7 +5,7 @@
     <section class="main-banner">
         <div class="container">
           <h1 class="fade-in-up one">
-            The Open Source
+            Open Source
```

Comment:
On 16 Mar 2015, at 13:27, Jirka Kremser wrote:

>> @@ -5,7 +5,7 @@
>>    <section class="main-banner">
>>        <div class="container">
>>          <h1 class="fade-in-up one">
>> -            The Open Source
>> +            Open Source
>
> (http://liveoak.io)

This was on purpose. It can't be "The", as there exist others like e.g. 
rhq.jboss.org

-- 
hwr@pilhuhn.de  m:0179/207 4919  b:http://pilhuhn.blogspot.com